src/bindings/python/tests/test_graph/test_ops_unary.py

```diff
@@ -8,13 +8,13 @@
 import openvino.runtime.opset9 as ov
 from openvino.runtime import Shape, Type
 from tests.runtime import get_runtime
-from tests.test_ngraph.util import run_op_node
+from tests.test_graph.util import run_op_node
 
 R_TOLERANCE = 1e-6  # global relative tolerance
 
 
 @pytest.mark.parametrize(
-    ("ng_api_fn", "numpy_fn", "range_start", "range_end"),
+    ("graph_api_fn", "numpy_fn", "range_start", "range_end"),
```

Comment:
In `graph_api_fn` the "fn" is presumably "function". Since the values given in tuples below are single operators, it could be renamed to `graph_api_op`.

Reply:
Done